Changelog.md

```diff
@@ -11,6 +11,7 @@ Bugfixes:
  * Commandline interface: Do not try creating paths ``.`` and ``..``.
  * Type system: Fix a crash caused by continuing on fatal errors in the code.
  * Type system: Disallow arrays with negative length.
+ * Inline assembly: Charge one stack slot for non-value types during analysis.
```

Comment:
(I don't know the layout of non-value types on the stack; so maybe this description is also correct)